ref/general/microshed.adoc

```diff
@@ -6,78 +6,87 @@
 :seo-description:  MicroShed testing helps you to write integration tests using testcontainers for Java microservice applications. With MicroShed testing you can test your Open Liberty application from outside the container so you are testing the exact same image that runs in production.
 = Testing in a container
 
-MicroShed testing helps you to write integration tests for Java microservice applications.
-With MicroShed testing, you reduce the risk of test failure by testing your application in the same container that you use in production.
+Microshed testing is a Java library that helps you write true-to-production integration tests for your microservice applications using testcontainers.
+While adhering to the standards of the development-production-parity for building modern applications, MicroShed Testing tests your application from outside in a black box fashion without accessing its internals.
+
+Integration testing evaluates the performance of a group of microservices as a whole unit, as compared to unit tests, which tests each microservice as an individual unit.
+Thus, integration tests are more comprehensive than unit tests.
+However, integration tests can be more complicated and take longer to run.
+Microshed testing makes your work easier and faster by streamlining integration tests with testcontainers.
 
-Integration testing looks at microservices as a group and tests them as a whole unit.
-Integration testing offers advantages over other types of testing, for example, detecting bugs earlier in the cycle.
 
 == Why integration testing
 
-Integration testing follows unit testing.
-After individual modules are unit tested, integration testing looks at the modules together to evaluate their performance as a group.
+You need to be sure that your application works in development as well as production mode.
```

Comment:
I added this sentence based on a presentation slide by Andrew which uses the word 'confident'. I used 'sure'. From a developer point of view a test can run in development but fail in production, so the developers needs to be sure/confident about their test in developement.

Reply:
ok- but I think the sentence construction is implying am emphasis on development rather than production, while this section emphasizes vulnerabilities to production. eg

You need to be sure that your applications works as well in production as it does in the development environment.